policyengine_us/parameters/gov/states/nj/tax/income/exemptions/blind_or_disabled/amount.yaml

```
@@ -0,0 +1,11 @@
+description: NJ provides a blind or disabled exemption of $1k for individuals and $2k for couples.
```

Descriptions omit values
```suggestion
description: New Jersey provides a blind or disabled exemption of this amount per filer or spouse.
```